src/GitHub.Exports/Primitives/UriString.cs

```diff
@@ -131,7 +134,9 @@ bool ParseScpSyntax(string scpString)
         /// <returns></returns>
         public Uri ToRepositoryUrl()
         {
-            if (url != null && IsFileUri) return url;
+            // we only want to process urls that represent network resources
+            if (!IsValidUri && !IsScpUri) return url;
+            if (IsValidUri && IsFileUri) return url;
```

Comment:
I think the previous two conditionals could be combined and simplified to:

`if (!IsScpUri && (!IsValidUri || IsFileUri))) return url;`